data/actionsmap/yunohost.yml

```diff
@@ -226,6 +226,97 @@ user:
                 username:
                     help: Username or email to get information
 
+#############################
+#           Alias            #
+#############################
+alias:
+    category_help: Manage alias
+    actions:
+
+        ### alias_list()
+        list:
+            action_help: List existing aliases
+            api: GET /alias
+            configuration:
+                authenticate: all
+                authenticator: ldap-anonymous
```

Comment:
ok. doable actually : alias_info and alias_list should *not* call _ensure_ldap_ou_is_created, and just gracefully exit if the ou is not created (alias_list should simply return an empty list, alias_info should output the same error as if the admin requested info on a non-existing alias).

thanks for catching this.